protocol-definitions/Metrics.yaml

```diff
@@ -0,0 +1,40 @@
+id: 39
+name: Metrics
+methods:
+  - id: 1
+    name: readMetrics
+    since: 2.0
+    doc: |
+      Reads the recorded metrics starting with the smallest sequence number
+      greater or equals to the sequence number set in fromSequence.
+    request:
+      retryable: false
+      acquiresResource: false
+      partitionIdentifier: name
+      params:
+        - name: uuid
+          type: String
+          nullable: false
+          since: 2.0
+          doc: |
+            The UUID of the member that is supposed to read the metrics from.
+        - name: fromSequence
```

Comment:
better naming suggestion: `startSequenceNumber`

Reply:
It is the sequence number that it will continue to read from.